lib/9pfs/9pfs.h

```diff
@@ -39,63 +39,150 @@
 
 #include <vfscore/prex.h>
 
-/*
- * Currently supports only the 9P2000.u variant of the protocol.
+/**
+ * Protocol version; currently only the `9P2000.u` variant of the protocol is supported
  */
 enum uk_9pfs_proto {
+	/*
+	 * Extensions and minor adjustments are added to
+	 * the basic protocol for Unix systems
+	 */
 	UK_9P_PROTO_2000U,
+	/*
+	 * Extensions and minor adjustments are added to
+	 * the basic protocol for Linux systems
+	 */
 	UK_9P_PROTO_2000L,
+	/*
+	 * The number of existent protocols;
+	 * in this case, two (`UK_9P_PROTO_2000U` and `UK_9P_PROTO_2000L`)
+	 */
 	UK_9P_PROTO_MAX
 };
 
+/**
+ * An entry containing the necessary data for mounting the filesystem
+ */
 struct uk_9pfs_mount_data {
-	/* 9P device. */
+	/* 9P device, used to interact with a `virtio` device;
+	 * the initialization for this field is done in
+	 * [`uk_9p_dev_connect`](https://github.com/unikraft/unikraft/blob/staging/lib/uk9p/9pdev.c#L237)
```

Comment:
```suggestion
	 * the initialization for this field is done in `uk_9p_dev_connect()`.
```
Just leave the name of the function (with `()`) at the end. `doxygen` knows to add links to the actual functions.